services/frontend-service/src/ui/components/chip/EnvironmentGroupChip.tsx

```diff
@@ -0,0 +1,123 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+import classNames from 'classnames';
+import { EnvPrio } from '../ReleaseDialog/ReleaseDialog';
+import { Environment } from '../../../api/api';
+import React from 'react';
+import { EnvironmentGroupExtended, useCurrentlyDeployedAtGroup } from '../../utils/store';
+import { Tooltip } from '@material-ui/core';
+import { Button } from '../button';
+import { LocksWhite } from '../../../images';
+
+export const EnvironmentChip = (props: {
+    className: string;
+    env: Environment;
+    groupNameOverride?: string;
+    numberEnvsDeployed?: number;
+    numberEnvsInGroup?: number;
+    withEnvLocks: boolean;
+}) => {
+    const { className, env } = props;
+    const priority = env.priority;
+    const priorityClassName = className + '-' + String(EnvPrio[priority]).toLowerCase();
+    const name = props.groupNameOverride ? props.groupNameOverride : env.name;
+    const numberString =
+        props.numberEnvsDeployed && props.numberEnvsInGroup
+            ? '(' + props.numberEnvsDeployed + '/' + props.numberEnvsInGroup + ')'
+            : '';
+    const locks = props.withEnvLocks ? (
+        <div className={classNames(className, 'env-locks')} style={{ backgroundColor: 'transparent', display: 'flex' }}>
```

Comment:
is this style really necessary?
```suggestion
        <div className={classNames(className, 'env-locks')}>
```